src/main/java/frc/robot/commands/SpinUpShooter.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class SpinUpShooter extends CommandBase {
+  
+  private ShooterSubsystem shooterSubsystem;
+
+  public SpinUpShooter() {
+    shooterSubsystem = ShooterSubsystem.getInstance();
+    addRequirements(shooterSubsystem);
+  }
+
+  @Override
+  public void initialize() {
+  }
+
+  @Override
+  public void execute() {
+    // Hardcoded RPM for testing
```

Comment:
maybe set a variable in the command that we temporarily set to 2000 RPM for testing, but eventually will be determined by a call to some kind of formula. Then refer to the variable on line 22 and 24.